modules/renter/uploadheap.go

```diff
@@ -62,7 +64,7 @@ func (uch *uploadChunkHeap) Pop() interface{} {
 	return x
 }
 
-// managedAddChunkToUploadHeap will add a chunk to the upload heap.
+// managedPush will add a chunk to the upload heap.
 func (uh *uploadHeap) managedPush(uuc *unfinishedUploadChunk) {
```

Comment:
@DavidVorick should be break out of this method once we realize that a file is not available offline and has less than 1x redundancy? At the very least I would add a `todo` for it because once we scan our used hosts every hour or so, we shouldn't end up in a state where we reach <1x redundancy due to the host being temporarily offline.

Reply:
@DavidVorick  thoughts?